core/opds2.py

```diff
@@ -176,14 +177,22 @@ def _contributors(self, edition: Edition) -> Dict:
 
     def feed_links(self):
         """Create links for a publication feed"""
-        next_query_string = (
-            f"{self.pagination.next_page.query_string}&{self.facets.query_string}"
-        )
-        next_url = self.url.split("?", 1)[0] + "?" + next_query_string
         links = [
-            {"href": self.url, "rel": "self", "type": self.OPDS2_TYPE},
-            {"href": next_url, "rel": "next", "type": self.OPDS2_TYPE},
+            {"href": cdnify(self.url), "rel": "self", "type": self.OPDS2_TYPE},
```

Comment:
How come you used `cdnify` here?

Reply:
Good question, I was comparing the OPDS1.2 feed code to this code (which I have written), the feed links use this method in case the production deployment has a cdn configured